client/blocks/domain-to-plan-nudge/index.jsx

```diff
@@ -80,9 +79,22 @@ class DomainToPlanNudge extends Component {
 		} );
 
 		page( `/checkout/${ siteId }/personal` );
+	};
+
+	renderRegular() {
+		const { siteId, translate } = this.props;
+		return (
+			<UpgradeNudge
+				title={ translate( 'Upgrade to a Personal Plan and Save!' ) }
+				message={ translate( 'Get live chat support and no ads bundled with your domain!' ) }
```

Comment:
"and no ads bundled": I think users could miss the "no" part here and also might not know what an "ad" is without seeing the "WP.com" before it. I would reword it into something like: "and remove WordPress.com Ads".